### PR TITLE
Have enqueued_at time be nil for jobs that have yet to be enqueued

### PR DESCRIPTION
**Pull request by jonhyman**
(i.e., are scheduled).

### CODE REVIEW

File: test/test_api.rb [hunk 1]
```diff
@@ -207,6 +207,13 @@ class ApiWorker
       assert_equal 0, q.size
     end
 
+    it 'has no enqueued_at time for jobs enqueued in the future' do
+      job_id = ApiWorker.perform_in(100, 1, 'foo')
+      q = Sidekiq::Queue.new
```
Comment (mperham) [inline test/test_api.rb]: Superfluous line.

File: web/locales/en.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ en: # <---- change this to your locale code
   Scheduled: Scheduled
   Retries: Retries
   Enqueued: Enqueued
+  NotYetEnqueued: Not yet enqueued
```
Comment (mperham) [inline web/locales/en.yml]: Put new locale strings at the bottom of the file.

File: web/views/_job_info.erb [hunk 3]
```diff
@@ -34,7 +34,7 @@
       </tr>
       <tr>
         <th><%= t('Enqueued') %></th>
-        <td><%= relative_time(job.enqueued_at) %></td>
+        <td><%= job.enqueued_at ? relative_time(job.enqueued_at) : t('NotYetEnqueued') %></td>
```
Comment (mperham) [inline web/views/_job_info.erb]: Use a local so we don't call enqueued_at and do the time math twice.

File: test/test_api.rb (hunk 1 above)
Comment (jonhyman) [inline test/test_api.rb]: copy and paste fail. :grin: updated

File: web/locales/en.yml (hunk 2 above)
Comment (jonhyman) [inline web/locales/en.yml]: sounds good

**Pull request merged**